File: terra/src/main/scala/com/dafttech/terra/game/world/entities/living/ai/AIControlled.java
package com.dafttech.terra.game.world.entities.living.ai;

import com.dafttech.terra.engine.input.InputHandler$;

public class AIControlled extends ArtificialIntelligence {
    @Override
    public void update(float delta) {
        if (InputHandler$.MODULE$.isKeyDown("LEFT")) assignedEntity.walkLeft();
        if (InputHandler$.MODULE$.isKeyDown("RIGHT")) assignedEntity.walkRight();
        if (InputHandler$.MODULE$.isKeyDown("JUMP") && !assignedEntity.isInAir()) assignedEntity.jump();
    }
}

File: terra/src/main/scala/com/dafttech/terra/game/world/entities/living/ai/AIControlled.scala
package com.dafttech.terra.game.world.entities.living.ai

import com.dafttech.terra.engine.input.InputHandler

class AIControlled extends ArtificialIntelligence {
  override def update(delta: Float): Unit = {
    if (InputHandler.isKeyDown("LEFT")) assignedEntity.walkLeft()
    if (InputHandler.isKeyDown("RIGHT")) assignedEntity.walkRight()
    if (InputHandler.isKeyDown("JUMP") && !assignedEntity.isInAir) assignedEntity.jump()
  }
}